### PR TITLE
Document addition of ref to GetAContactResponse

### PR DESCRIPTION
**Pull request by zubin**
See splitpayments/split/pull/5461

### CODE REVIEW

File: source/openapi3/split.yaml [hunk 1]
```diff
@@ -4508,6 +4513,7 @@ components:
       example:
         data:
           id: 55afddde-4296-4daf-8e49-7ba481ef9608
+          ref: oustanding-tours
```
Comment (corroded) [inline source/openapi3/split.yaml]: Contact refs start with a CNT (unfortunately 😂 ) so it should be `CNT.123`

**Review by corroded — APPROVED** [review]
Happy for this to get merged once comment is addressed 👍

File: source/openapi3/split.yaml (hunk 1 above)
Comment (zubin) [reply source/openapi3/split.yaml]: Updated: 44e08a2

**Comment by zubin** [comment]
Converted to draft until splitpayments/split/pull/5461 is deployed so it doesn't get merged accidentally.

**Pull request merged**